cli/lib/kontena/plugin_manager.rb

```diff
@@ -56,6 +56,8 @@ def uninstall_plugin(plugin_name)
     # Search rubygems for kontena plugins
     # @param pattern [String] optional search pattern
     def search_plugins(pattern = nil)
+      require 'excon'
+      require 'json'
```

Comment:
Have to say I find it weird to have `require` on method level. This require juggling starts to be a slippery slope now if we start to mangle all requires on method level "just-in-case" and to optimise cli load time.